### PR TITLE
Add aid for thread messages and added test for checking thread rendering

### LINKED ISSUE #959: ui tests for thread rendering
- test that date of last message in the thread is displayed on each inbox item
 - make sure that messages are ordered by this date in inbox, with threads that have the most recent message on top

### LINKED ISSUE #1370: add new accessibility id for thread messages
__Originally posted by @fcvakintos in https://github.com/FlowCrypt/flowcrypt-ios/issues/959#issuecomment-1032448491__

### PR DESCRIPTION
**Pull request by Kharchevskyi**
This PR adds accessibility ids and add test for checking thread rendering, updated test after pushing new aids, added new methods for checking thread

close #1370  
close #959 
----------------------------------

**Tests** 
- Tests added or updated

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [x] addresses the issue it closes (if any)
- [x] code is readable and understandable
- [x] is accompanied with tests, or tests are not needed
- [x] is free of vulnerabilities
- [x] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

**Comment by fcvakintos** [comment]
@tomholub I will fix the test on my branch

**Comment by fcvakintos** [comment]
@tomholub ok, will continue on this branch

**Comment by tomholub** [comment]
The git history seems unhappy - we will need to do something about it before merging. Maybe squashing the history first. Otherwise not possible to review.

**Review by tomholub — APPROVED** [review]

File: appium/tests/specs/live/inbox/CheckThreadRendering.spec.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    const dateThird = CommonData.threadMessage.thirdDate;
+
+    await SplashScreen.login();
+    await SetupKeyScreen.setPassPhrase();
+    await MailFolderScreen.checkInboxScreen();
+
+    await MailFolderScreen.clickOnEmailBySubject(emailSubject);
+    await EmailScreen.checkThreadMessage(senderEmail, emailSubject, thirdMessage, dateThird, 2);
+    await EmailScreen.checkThreadMessage(userEmail, emailSubject, secondMessage, dateSecond, 1);
+    await EmailScreen.checkThreadMessage(senderEmail, emailSubject, firstMessage, dateFirst);
```
[the remark this answers is not in the document]
Comment (fcvakintos) [reply appium/tests/specs/live/inbox/CheckThreadRendering.spec.ts:29]: it works on local and romote

**Pull request merged**